mobile/src/Student/screens/StudentGameIntro/IntroInfo/index.js

```diff
@@ -1,12 +1,18 @@
 import React from 'react'
 import { StyleSheet, Text, View, Image, Dimensions } from 'react-native'
+import { scale, verticalScale } from "react-native-size-matters"
 import { fontFamilies, fonts, colors } from '../../../../utils/theme'
 const win = Dimensions.get('window');
 
-const IntroInfo = ({ image, text }) => {
+const IntroInfo = ({ screen, Monsters, phone, sticker, text }) => {
     return (
         <View style={styles.container}>
-            <Image source={image} style={styles.image} />
+            <Image source={Monsters[0]} style={screen === 1 ? styles.monster1 : styles.monster3} />
```

Comment:
using screen to handle the styles to be used here kinda make this component to be tightly coupled with its parent and not reusable for any other cases. Probably a better design can be passing the styles instead of `screen` property so this component can just use them.